conn.go

```diff
@@ -16,7 +18,7 @@ func (c *conn) IsClosed() bool {
 }
 
 // OpenStream creates a new stream.
-func (c *conn) OpenStream() (mux.MuxedStream, error) {
+func (c *conn) OpenStream(context.Context) (mux.MuxedStream, error) {
```

Comment:
mplex actually uses a context: https://github.com/libp2p/go-mplex/blob/master/multiplex.go#L322-L328 We should percolate it down.

Reply:
mplex has a hardcoded timeout right now. The context should override that timeout.

Reply:
@raulk Opened #22 for that.